server/utils/index.js

```diff
@@ -0,0 +1,22 @@
+export default {
+
+  normalizeData(data) {
+    if (!data) {
+      return [];
+    }
+    const keys = Object.keys(data);
+    const result = [];
+    keys.forEach((key) => {
```

Comment:
You should replace this forEach here with map. You are essentially returning an array at the end of the day